addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -8409,6 +8416,75 @@ registry.VersionControl = SnippetOptionWidget.extend({
         });
         return this._super(...arguments);
     },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async updateUI() {
+        // Waiting for the options to be visible so the alert can be correctly
+        // displayed.
+        this.options.wysiwyg.waitForEmptyMutexAction().then(() => {
```

Comment:
I think this is not needed (maybe will not be after `_renderCustomXML` change?). Or maybe what you want here is using `this.$overlay.data('$optionsSection')[0]` (it's a bit hacky but it's the correct way to access the options container you are looking for I think (you could not have guessed it ^^)).